src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/EdgeArchProxyIT.java

```diff
@@ -134,7 +132,7 @@ public void shouldInjectIndividualizedPushPromisesOnSharedFreshnessExtension() t
     {
         k3po.finish();
         counters.assertExpectedCacheEntries(1);
-        counters.assertRequestsSlots(0);
+        counters.assertRequestsSlots(1);
```

Comment:
Would it be better to adjust the script to cleanup the push promise request at the end, so that the request slots in use at the end remains `0`?

Reply:
since now we are properly cleaning up request slot and the script is sending abort frame we should set it back to `0`